### ISSUE TITLE
Cannot find shearwater peregrine on Android

### ISSUE THREAD

**Issue by vondwater** [subsurface/subsurface#2974]
### Describe the issue:

- [x ] Bug

### Issue long description:
My Shearwater Peregrine is not found on Android. I have already tried to turn on 'Show all bluetooth devices' toggle in the Advanced settings but still don't see my DC.

### Operating system:
Android

### Subsurface version:
Subsurface version 3.0.15 (4.9.6.111) official release

### Steps to reproduce:
1) Turn on bluetooth connection on DC
2) Try to find DC in subsurface

### Current behavior:
My DC is not found in the app

### Expected behavior:
DC is found in the app

**Comment by dirkhh** [comment]
Please use a free app like nRF Connect to connect to and bond with the dive computer.
Then start Subsurface-mobile again. I am 99.9% sure that things will simply start working for you.

**Comment by vondwater** [comment]
Thanks for your answer. I have tried to do that but the perigrine is not bondend. How can I bond?
![Screenshot_20200925-122434](https://user-images.githubusercontent.com/841864/94256681-71223d00-ff2a-11ea-91d2-37c146e972a3.jpg)

**Comment by dirkhh** [comment]
click on the three dots next to 'DISCONNECT' near the top right and the option will be in that menu.

BLE is unfortunately a very poorly spec'ed protocol. Let me rephrase that - there are great specs, but they leave out a lot of things that would have been important to define. As a result, every dive computer implementing BLE is slightly different and in the end that makes our lives harder. Subsurface-mobile on Android currently isn't looking for BLE devices itself, it relies on the OS telling us about them. That is, honestly, a design flaw that we need to fix. And it's the reason for this odd requirement to bond the dive computer first (that way it will be listed by the OS when we look for it).

Not ideal, but this is a one time process and after that things should "just work" (famous last words)

**Comment by vondwater** [comment]
I have tried to do what you wrote but I cannot bond with my peregrine with the app. Is there any other possibility?

**Comment by dirkhh** [comment]
Frustratingly this worked a couple of weeks ago, and now it fails for me as well.
Back to the drawing board - I hope I'll have a solution for this soon.
